react-main/src/redux/profile/actions.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { push } from 'react-router-redux';
+
+export const actions = {
+  PROFILE: 'PROFILE'
+};
+
+export const profile = () => ({ type: actions.PROFILE });
```

Comment:
why do you need this action? No reducer is listening to it, and it has no logic. Remove it. And remove this file altogether, as it has no other actions